implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -104,8 +106,14 @@ pub struct NodeManager {
 }
 
 impl NodeManager {
-    pub(super) fn identity(&self) -> Identity {
-        self.identity.clone()
+    pub(super) async fn identity(&self) -> Result<Identity> {
```

Comment:
Should this function exist? That allows us to forget to fetch the latest Identity instance or e.g. rotate key using this cloned struct which will have no effect on the original Identity

Reply:
This should be well documented at least

Reply:
Correct, it is not needed, I removed it.